internal/storage/util.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021-2024 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package storage
+
+import (
+	"context"
+
+	"github.com/cerbos/cerbos/internal/policy"
+)
+
+const MaxPoliciesInBatch = 5
```

Comment:
```suggestion
const MaxPoliciesInBatch = 25
```

We can set this a little higher. The batch size has to be reasonably sized to see any benefits from batching.